frontend/src/components/translation/TranslationTable.tsx

```diff
@@ -66,13 +132,54 @@ const TranslationTable = ({
           </Text>
         )}
         <Flex direction="column" width="130px" margin="10px">
-          <Badge
-            textTransform="capitalize"
-            variant={getStatusVariant(storyLine.status)}
-            marginBottom="10px"
-          >
-            {storyLine.status}
-          </Badge>
+          {!editable ? (
+            <Menu>
+              <MenuButton
+                as={Badge}
+                textTransform="capitalize"
+                variant={getStatusVariant(storyLine.status)}
+                marginBottom="10px"
+              >
+                {storyLine.status}
+                <ChevronDownIcon />
+              </MenuButton>
+              <MenuList>
+                <MenuItem
+                  icon={<CheckIcon />}
+                  onClick={async () => {
+                    handleStatusChange(storyLine, "APPROVED");
+                  }}
+                >
+                  Approve
+                </MenuItem>
+                <MenuItem
+                  icon={<WarningTwoIcon />}
+                  onClick={async () => {
+                    handleStatusChange(storyLine, "ACTION_REQUIRED");
+                  }}
+                >
+                  Action Required
+                </MenuItem>
+                <MenuItem
+                  icon={<TimeIcon />}
+                  onClick={async () => {
+                    handleStatusChange(storyLine, "DEFAULT");
+                  }}
+                >
+                  Pending
+                </MenuItem>
+              </MenuList>
+            </Menu>
+          ) : (
+            <Badge
+              textTransform="capitalize"
+              variant={getStatusVariant(storyLine.status)}
+              marginBottom="10px"
+            >
+              {storyLine.status}
+            </Badge>
+          )}
+          <Button variant="addComment">Comment</Button>
```

Comment:
Is this an extra button for leaving comments? there's one on 184 that has actual functionality. I think this should be removed as it appears useless